README.md

```diff
@@ -107,11 +107,18 @@ pip install -e ".[dev]"
 ```
 and follow the more detailed instructions in [CONTRIBUTING](https://github.com/facebookresearch/theseus/blob/main/CONTRIBUTING.md).
 
+Compilation from source supports is affected by the following enviroment variables:
+* `BASPACHO_ROOT_DIR`: root dir of [BaSpaCho](https://github.com/facebookresearch/baspacho) sparse solver, which must be already
+  built with binaries in the subdirectory `build`.
+* `THESEUS_ENABLE_CUDA` (1 or 0): if set, forces enabling or disabling Cuda support regardless of system's `torch.cuda.is_available()`.
```

Comment:
Can we make this easier for users?

Can we do `pip install -e .` like before and cpu baspacho is installed while gpu baspacho is installed if cuda is available just like the current extlib stuff.

For these env variables can we add reasonable defaults in setup.py that could overridden from command line with a flag after `pip install -e .` if needed.

Reply:
Not entirely sure about this - there are some points which are a bit delicate, in particular how to compile baspacho so that it takes advantage of system BLAS if available, and reverts to a fallback if not available. Also, it may fail if CMake is not recent enough, and if GCC/NVCC are not compatible versions. So I would consider reasonable to leave the user control over how it's installed, for the time being. Maybe we can start like this and then when confident that there is no installation issues we make setup.py install it automatically (it would be at least a bit of work required to achieve this too I think  right? @luisenp )

Reply:
renamed.